opm/simulators/flow/BlackoilModelParameters.cpp

```diff
@@ -276,6 +279,9 @@ void BlackoilModelParameters<Scalar>::registerParameters()
 
     Parameters::Hide<Parameters::DebugEmitCellPartition>();
 
+    Parameters::Register<Parameters::GroupMaxIterations>
+    ("Maximum number of iterations in the group solver algorithm");
```

Comment:
The iteration not only handles the group control, when a well has multiple constraints itself, an iteration process might also be performed (also it is not that of a problem compared with group or network related).  We need to get a better  or more correct name?

Reply:
What about WellGroupConstraintsMaxIterations?

Reply:
It is better and usable. We can use it before we find something else.